src/main/java/com/google/sps/servlets/CronServlet.java

```diff
@@ -49,11 +50,13 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
         // Clears the Daily Deed Property for all elements
-        resetDailyDeed();
+        resetDailyDeed(datastore);
 
         // Pulls all Database Entries
-        List<GoodDeed> GoodDeeds = PullDeedsFromDatastore();
+        ArrayList<GoodDeed> GoodDeeds = PullDeedsFromDatastore(datastore);
```

Comment:
just fyi - always a good idea to have _camelCase_ for local variables and method names. This is just a matter of convention